### PR TITLE
fix(rest/queue): logical error in response status handling

### LINKED ISSUE #546: Response status checking for readable errors will never be true
Relevant code: https://github.com/discordeno/discordeno/blob/997189ec4af56a362c6460471d808a7ad9c2a574/src/rest/queue.ts#L65

I believe this should be a `||` (though `< 200` does not feel necessary, given 1xx http codes are not *technically* an error).

### PR DESCRIPTION
**Pull request by ayntee**
Closes #546

### CODE REVIEW

File: src/rest/queue.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export async function processQueue(id: string) {
         response.headers,
       );
 
-      if (response.status < 200 && response.status >= 400) {
+      if (response.status < 200 || response.status >= 400) {
```
Comment (itohatweb) [inline src/rest/queue.ts:65]: I like this more if you have nothing against changing it.
```suggestion
      if (response.status < 200 || 400 <= response.status) {
```

**Review by itohatweb — APPROVED** [review]
Then merge it

**Comment by ayntee** [comment]
> Then merge it

image

**Comment by itohatweb** [comment]
I guess Skillz needs to approve it

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**